sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -91,6 +91,11 @@ pub struct BootstrapOptions {
         help = "Bootstrap protocol to use"
     )]
     pub protocol: BootstrapProtocol,
+    #[structopt(
+        long,
+        help = "Whether to reset target and clear UART RX buffer after bootstrap. For CW310 only."
+    )]
+    pub no_clear_uart_rx: Option<bool>,
```

Comment:
I didn't notice this during the review, but this has an unintended side effect: it is no longer possible to leave a console open and run a bootstrap at the same time unless one specifies --no-clear-uart-rx=true.

I'd suggest we change this:
1. Negative-named options are evil.  This should be `--clear-uart-rx`.
2. This should default to false (ie: don't touch the uart unless asked).
3. We should change the test commands in `opentitan.bzl` to use `--uart-clear-rx` so we get the desired behavior when running tests.